### PR TITLE
DM-10430: Add time stamps to the console logs

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: config/log4cxx.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@ log4j.rootLogger=INFO, A1, F1
 log4j.appender.A1=ConsoleAppender
 log4j.appender.A1.Target=System.err
 log4j.appender.A1.layout=PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%X{PID} %-5p %c: %m%n
+log4j.appender.A1.layout.ConversionPattern=%X{PID} %-5p %d{yyyy-MM-ddThh:mm:ss.sss} %c: %m%n
```
Comment (PaulPrice) [inline config/log4cxx.properties]: This looks reasonable.
What timezone is used? If UTC, you should add a `Z` at the end of the time format.

Comment (hsinfang) [inline config/log4cxx.properties]: They would be the same time zone as the machines.  Would that be fine?

Comment (PaulPrice) [inline config/log4cxx.properties]: That's great, thanks!

Comment (ktlim) [inline config/log4cxx.properties]: I would really like the timezone to be logged in all cases.  Can you add a "Z" (which is a date formatter character, and only coincidentally becomes a UTC indicator) to the end of this?

Comment (hsinfang) [inline config/log4cxx.properties]: Opened https://github.com/lsst/ctrl_pool/pull/16 

I assume we don't need another ticket?

Comment (hsinfang) [inline config/log4cxx.properties]: https://github.com/lsst/ctrl_pool/pull/16 has been approved and merged.

**Pull request merged**